test/test_functional_tensor.py

```diff
@@ -64,22 +64,24 @@ def test_hflip(self):
 
     def test_crop(self):
         script_crop = torch.jit.script(F_t.crop)
```

Comment:
Do we want to change this to be `F.crop` ?

Reply:
Sounds good, I can change it in #2569